server/src/alerts.rs

```diff
@@ -16,16 +16,18 @@
  *
  */
 
+use std::sync::atomic::{AtomicU32, Ordering};
+
 use log::{error, info};
 use serde::{Deserialize, Serialize};
 
-#[derive(Default, Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Default, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Alerts {
     pub alerts: Vec<Alert>,
 }
 
-#[derive(Default, Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Alert {
     pub name: String,
```

Comment:
As we discussed, let's add a system generated `id` here apart from alert name. While saving this to S3, let's use the `id` as the file name.